src/main/java/org/f24/service/validator/impl/FormValidator.java

```diff
@@ -50,26 +53,50 @@ public void validate() throws ProcessingException, IOException, ResourceExceptio
         }
 
         validateTaxCode();
+        validateIdCode();
```

Comment:
Intendevo la riga di FormValidator  dove si fa: `ObjectMapper objectMapper = new ObjectMapper(); ` (effettivamente solo in questa classe si usa ObjectMapper). Puoi utilizzare sempre la stessa istanza di ObjectMapper, invece di creare una nuova istanza per ogni invocazione del metodo **validate()** (puoi creare un campo final di ObjectMapper)

Reply:
Abbiamo integrato la segnalazione